packages/ui-components/src/TextArea.tsx

```diff
@@ -6,21 +6,19 @@ import React from 'react';
 import { Form } from 'semantic-ui-react';
 import SUITextArea from 'semantic-ui-react/dist/commonjs/addons/TextArea';
 
-type Props = {
+export type TextAreaProps = {
```

Comment:
I don't personally a need to export this?

Reply:
I was maybe anticipating a scenario that won't happen:

```typescript
import { BalanceDisplay, BalanceDisplayProps} from '@substrate/ui-components';

const MyBalance = styled<BalanceDisplayProps>(BalanceDisplay)`
  // some css stuff
`;
```

anyways, removed